robottelo/config/base.py

```diff
@@ -644,7 +645,7 @@ def read(self, reader):
     def validate(self):
         """Validate rhev settings."""
         validation_errors = []
-        values = [v for k, v in vars(self).iteritems() if k != 'ca_cert']
+        values = [v for k, v in iteritems(vars(self)) if k != 'ca_cert']
```

Comment:
Maybe `import six` and then `six.iteritems(vars(self))` to avoid any confusion with python's iteritems?

Reply:
hmm, that module already uses few like `from six.  ` , so thought I would follow the same.